core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -71,6 +69,9 @@ class SparkEnv (
     val outputCommitCoordinator: OutputCommitCoordinator,
     val conf: SparkConf) extends Logging {
 
+  // We set the ShuffleManager in SparkContext and Executor
```

Comment:
nit update comment to say something like: the ShuffleManager is initialized later in... to allow it being defined in user specified jars.